src/starbase/config.ts

```diff
@@ -82,18 +101,49 @@ function validateStarbaseConfigSchema(config: StarbaseConfig) {
   };
 
   const validator = ajv.compile(integrationSchema);
+  const configValidator = ajv.compile(configSchema);
+  const storageValidator = ajv.compile(storageSchema);
+
+  let configErrorCount = 0;
+
+  if (!configValidator(config)) {
+    console.error(
+      `ERROR:  config file validation error(s):  `,
+      configValidator.errors,
+    );
+    configErrorCount++;
+  }
+
+  if (!storageValidator(config.storage)) {
+    console.error(
+      `ERROR:  storage configuration validation error(s):  `,
+      storageValidator.errors,
+    );
+    configErrorCount++;
+  }
 
   for (const integration of config.integrations) {
+    const integrationName =
+      integration.name || integration.instanceId || 'Unknown integration';
```

Comment:
NOTE: `Unknown integration` is being used because the schema validation could fail and both of these integration properties could be undefined.